src/coffea/nanoevents/methods/nanoaod.py

```diff
@@ -303,19 +303,29 @@ class Photon(candidate.PtEtaPhiMCandidate, base.NanoCollection, base.Systematic)
     def mass(self):
         return 0.0 * self.pt
 
+    @property
+    def charge(self):
+        return 0.0 * self.pt
+
     @property
     def isLoose(self):
         """Returns a boolean array marking loose cut-based photons"""
+        if "cutBased" in self.fields:
```

Comment:
Can you add a note that this is for v9 +

Reply:
Yup will do. I’m chatting a bit with E/Gamma ID if this is the correct way to access the bitmask. I’m surprised a bit because it returns zero tight photons over like a half a million event file.